README.md

```diff
@@ -12,11 +12,11 @@ rav1e is an AV1 video encoder. It is designed to eventually cover all use cases,
 * 64x64 superblocks
 * 4x4 to 64x64 RDO-selected square and 2:1/1:2 rectangular blocks
 * DC, H, V, Paeth, smooth, and a subset of directional prediction modes
-* DCT, ADST and identity transforms (up to 64x64, 16x16 and 32x32 respectively)
+* DCT, ADST, and identity transform (up to 64x64, 16x16 and 32x32 respectively)
```

Comment:
I believe this was correct before.  It's referring to multiple transforms, so it should be plural.  I don't disagree with the added comma, though; I prefer an Oxford comma myself.